lib/fxpay/settings.js

```diff
@@ -10,6 +10,9 @@
     //
     // Disallow receipts belonging to other apps.
     allowAnyAppReceipt: false,
```

Comment:
Is there ever a good reason to have this setting set to true? Should we log something in all caps if it's true? :)